### ISSUE TITLE
mach should detect aarch64 cpu type

### ISSUE THREAD

**Issue by cosmo0920** [servo/servo#18217]
Currently, mach does not detect aarch64 cpu_type.
I'm servo newbie, so I have no idea about this is an real servo build tool issue.

### Environment

* Hardware: Jetson TX2
* OS: Ubuntu 16.04.3 LTS  aarch64

### CPU Type in Python

```python
Python 2.7.12 (default, Nov 19 2016, 06:48:10)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.machine().lower()
'aarch64'
```

### Current implementation

Does not handle aarch64 cpu_type:
https://github.com/servo/servo/blob/master/python/servo/util.py#L31

### Proposed fix

https://github.com/cosmo0920/servo/commit/4a9b4dfb69ea523c6a1a68a25e7fc2e602482e94

**Comment by jdm** [comment]
Huh, I guess we have always done cross-compiles for aarch64 before. That seems like a fine change to me, if it works for you.

**Comment by cosmo0920** [comment]
Yeah, above patch works for me.
But this is one of work to build servo on aarch64.

**Comment by atouchet** [comment]
Should this be closed now with #18220 merged?
